sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -720,6 +720,15 @@ object RapidsConf {
       .createWithDefault(false)
 
   // FILE FORMATS
+  val MULTITHREAD_READ_NUM_THREADS = conf("spark.rapids.sql.multiThreadedRead.numThreads")
+      .doc("The maximum number of threads on each executor to use for reading small " +
+        "files in parallel. This can not be changed at runtime after the executor has " +
+        "started. Used with COALESCING and MULTITHREADED readers, see " +
+        "spark.rapids.sql.format.parquet.reader.type for a discussion of reader types.")
```

Comment:
this is referencing the parquet reader type still, probably want to add orc and avro or make it more generic somehow, that might help user know which reader types it works with as well